src/components/Editor/SearchBar.js

```diff
@@ -107,6 +107,7 @@ const SearchBar = React.createClass({
   componentDidMount() {
     // overwrite searchContents with a debounced version to reduce the
     // frequency of queries which improves perf on large files
+    // $FlowIgnore
     this.searchContents = debounce(this.searchContents, 100);
```

Comment:
Hi @jasonLaster 

Here I disable `flow` to suppress the error. Are we okay with this approach?
<img width="934" alt="screen shot 2017-03-27 at 3 16 43 pm" src="https://cloud.githubusercontent.com/assets/4562118/24345686/d79a652e-1303-11e7-87ab-09874cf52204.png">